From f5c4721500ea33adcae30dc8700efe8dd643a4a9 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sun, 12 Nov 2023 17:22:39 -0600
Subject: [PATCH] Don't run ldconfig on make install

No idea why it suddenly started doing this, but it's failing because it doesn't have permission to write to /etc. Don't let it do it.

---
 lib/Makefile.elf-lib | 2 --
 1 file changed, 2 deletions(-)

diff --git a/lib/Makefile.elf-lib b/lib/Makefile.elf-lib
index f850f3dd..ea0bbde8 100644
--- a/lib/Makefile.elf-lib
+++ b/lib/Makefile.elf-lib
@@ -50,8 +50,6 @@ install-shlibs install:: $(ELF_LIB) installdirs-elf-lib $(DEP_INSTALL_SYMLINK)
 	$(E) "	SYMLINK $(libdir)/$(ELF_IMAGE).so"
 	$(Q) $(INSTALL_SYMLINK) $(ELF_INSTALL_DIR)/$(ELF_SONAME) \
 		$(libdir)/$(ELF_IMAGE).so $(DESTDIR)
-	$(E) "	LDCONFIG"
-	$(Q) -$(LDCONFIG)
 
 install-strip: install
 	$(E) "	STRIP-LIB $(ELF_INSTALL_DIR)/$(ELF_LIB)"
-- 
2.42.0

